infrastructure/constructs/task_job_definition.py

```diff
@@ -24,9 +26,19 @@ def __init__(
         else:
             batch_job_definition_memory_limit = 500
 
+        check_output(
```

Comment:
This should probably be `check_call` since we're not doing anything with the output.